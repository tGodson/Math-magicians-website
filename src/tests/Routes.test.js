import React from 'react';
import renderer from 'react-test-renderer';
import Link from '../Routes';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="/">Home</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="/calculator">App</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="/quote">Quote</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});