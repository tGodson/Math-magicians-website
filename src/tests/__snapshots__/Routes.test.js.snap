// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav>
    <a
      aria-current="page"
      className="navlink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
  <hr />
  <div>
    <h1>
      Math Magician
    </h1>
    <h2>
      Welcome to out Page!
    </h2>
    <p>
      Hello, we are glad to introduce you to the Math Magician website. on this website, you can navigate to the calculatore page or to the quote page
    </p>
    <p>
      Have fun and thank you for comming!
    </p>
  </div>
</div>
`;

exports[`renders correctly 2`] = `
<div>
  <nav>
    <a
      aria-current="page"
      className="navlink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
  <hr />
  <div>
    <h1>
      Math Magician
    </h1>
    <h2>
      Welcome to out Page!
    </h2>
    <p>
      Hello, we are glad to introduce you to the Math Magician website. on this website, you can navigate to the calculatore page or to the quote page
    </p>
    <p>
      Have fun and thank you for comming!
    </p>
  </div>
</div>
`;

exports[`renders correctly 3`] = `
<div>
  <nav>
    <a
      aria-current="page"
      className="navlink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      aria-current={null}
      className="navlink"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
  <hr />
  <div>
    <h1>
      Math Magician
    </h1>
    <h2>
      Welcome to out Page!
    </h2>
    <p>
      Hello, we are glad to introduce you to the Math Magician website. on this website, you can navigate to the calculatore page or to the quote page
    </p>
    <p>
      Have fun and thank you for comming!
    </p>
  </div>
</div>
`;
